tests/popup-test.js

```diff
@@ -72,40 +77,34 @@ describe('addToDict(tempScript, htmlString, foundThis, key, category)', function
 });
 
 
-const loadJSON = (callback) => {
-    let xobj = new XMLHttpRequest();
-    xobj.overrideMimeType("application/json");
-    xobj.open('GET', 'my_data.json', true);
-    // Replace 'my_data' with the path to your file
-    xobj.onreadystatechange = () => {
-        if (xobj.readyState === 4 && xobj.status === "200") {
-            // Required use of an anonymous callback 
-            // as .open() will NOT return a value but simply returns undefined in asynchronous mode
-            callback(xobj.responseText);
-        }
+describe('vendors.json should be valid json', function () {
+
+  it('should be a valid file', function () {
+    
+    const vendorSchema = {
+      'id': '/SimpleVendors',
+      'type': 'object',
+      'properties': {
+        'regex': {
+          'type': 'array',
+          'items': {
+            'type': 'string'
+          }
+        },
+        'tooltip': {
+          'type': 'string'
+        },
+        'category': {
+          'type': 'string'
+        },
+      },
+      'required': ['category', 'regex']
     };
-    xobj.send(null);
-}
 
+    v.addSchema(vendorSchema, '/SimpleVendors');
+    const result = v.validate(vendors.vendor, vendorSchema);
 
-describe('vendors.json should be valid json', function () {
-  
-  beforeEach(() => {
-    sinon.stub(window, 'fetch');
-  });
-  
-  afterEach(() => {
-    window.fetch.restore();
-  });
-  
-  it('should be a valid file', function () {
-    fetch('../readiness-tool/vendors')
-      .then(function(response) {
-        console.log('response', response);
-        console.log('response text', response.text);
-
-      }).catch(function(err) {
-          // Error :(
-      });
+    expect(isJSON(vendors)).to.be.true;
```

Comment:
Not sure this is necessary. Looks like internally this just uses `JSON.stringify` and `JSON.parse`: https://github.com/shubhamoli/is-valid-json/blob/master/is-valid-json.js